### ISSUE TITLE
Error when parsing CSR from PEM with an IP as a SAN

### ISSUE THREAD

**Issue by Will-Low** [est31/rcgen#120]
I'm trying to parse a CSR from PEM. The PEM has an IP address in the SAN.

When I parse the PEM, I'm getting an "InvalidNameType" error. Is there a reason this shouldn't be permitted?

Example of what can result in this condition:
```rust
        let ip_addr = IpAddr::from_str("127.0.0.1").unwrap();

        let mut cert_params = CertificateParams::new([]);
        cert_params.subject_alt_names = vec![SanType::IpAddress(ip_addr)];

        let cert = rcgen::Certificate::from_params(cert_params).unwrap();

        let csr_pem = cert.serialize_request_pem().unwrap();

        // This will panic on upwrap due to "InvalidNameType"
        let csr = CertificateSigningRequest::from_pem(&csr_pem).unwrap();
```

**Comment by est31** [comment]
Can you try using rcgen from git? I think this has been addressed by c2bcdbf33aed48cd41874368bccb174f70e92182 . It's not on crates.io yet, but I might do a release in the coming days.

**Comment by Will-Low** [comment]
Confirming that this appears to have been addressed. Thanks for the info!
